nasa/ow_autonomy
# Oceanwater 608 update fault autonomy parameters

ticket: [#608 ](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-608)

Summary:
* adding parameters for power 
* created Union of maps function to allow autonomy to track multiple faults that might impact distinct moves. 
**  i.e { Op_PanAntenna, impacted by either (AntennaFaults, PowerFaults) }
** i.e   { Op_Stow, impacted by either (ArmFaults, PowerFaults) },

Test:
* run any roslaunch
* select any power or arm fault
*  run autonomy plexil plan (any should work, but for arm and power ex you can try Unstow.plx, for power and antenna you can try TestAntennaCamear.plx)
** roslaunch ow_autonomy autonomy_node.launch plan:=Unstow.plx
** see output message of power or arm fault and move fail.
